.github/workflows/crowdin.yml

```diff
@@ -23,10 +18,15 @@ jobs:
     steps:
       - name: Checkout
         uses: actions/checkout@v3
+        with:
+          ref: lts-2025
```

Comment:
Same as https://github.com/nuxeo/nuxeo-coldstorage/pull/309#discussion_r1987516358. Also, using `github` context variabkles wouldn't work here in the case of a workflow called by another one.